src/plugins/dashboard/public/application/utils/use/use_dashboard_container.tsx

```diff
@@ -126,44 +132,66 @@ const createDashboardEmbeddable = (
 
   const getShouldShowEditHelp = () => {
     return (
-      !savedDash.panels.length &&
-      savedDash.viewMode === ViewMode.EDIT &&
+      !appState.getState().panels.length &&
```

Comment:
Is `appState.getState()` expensive enough to warrant storing in a variable and not call it repeatedly?